packages/react/src/components/Notification/Notification.tsx

```diff
@@ -216,8 +295,93 @@ NotificationIcon.propTypes = {
   notificationType: PropTypes.oneOf(['inline', 'toast']).isRequired,
 };
 
+/**
+ * ToastNotification
+ * =================
+ */
+
+export interface ToastNotificationProps {
+  /**
```

Comment:
should this extend Div attributes? extra props can be passed in through `..rest`